src/js/engagement_view/src/services/fetch.ts

```diff
@@ -1,4 +1,4 @@
-export const apiFetch = async (urlSlug: string, method = "GET") => {
+export const apiFetchWithBody = async (urlSlug: string, method = "GET") => {

```

Comment:
A) There's no "body" here, hence `withBody` doesnt make sense
B) There's a few places where we 
apiFetchWithBody
with POST not GET - in which case - why not just apiPostRequestWithBody in those cases?

Reply:
Fixed